exercises/high-scores/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+#[derive(Debug)]
+pub struct HighScores;
+
+impl HighScores {
+    pub fn new(_: &[u32]) -> Self {
```

Comment:
You refer to the `scores` variable below, but you haven't set it yet.

```suggestion
    pub fn new(scores: &[u32]) -> Self {
```

Reply:
Oops, saw the "use more placeholders" recommendation and forgot that it's used. Good catch!